src/gridstack.ts

```diff
@@ -87,6 +89,16 @@ export class GridStack {
     }
     return el.gridstack
   }
+  /**
+   * Will return i-th DDPlusgin registerd in GridStackDD
+   * @param i i-th plugin (default to 0)
+   *
+   * @example
+   * let GridStackDDJQueryUI = GridStack.getDDPlugin(0);
+   */
+  public static getDDPlugin(i = 0) {

```

Comment:
I think a better solution now is to have the user explicitly include the one they want instead of loading them both - which will auto-register.